src/frontend/apps/standalone_site/src/features/Contents/features/Video/components/Create/VideoCreateForm.spec.tsx

```diff
@@ -0,0 +1,328 @@
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import fetchMock from 'fetch-mock';
+import { createMemoryHistory } from 'history';
+import {
+  modelName,
+  UploadManagerStatus,
+  useUploadManager,
+} from 'lib-components';
+import { render, Deferred } from 'lib-tests';
+import { Router } from 'react-router-dom';
+
+import VideoCreateForm from './VideoCreateForm';
+
+const playlistsResponse = {
+  count: 1,
+  next: null,
+  previous: null,
+  results: [
+    { id: 'some-playlist-id', title: 'some playlist title' },
+    { id: 'an-other-playlist', title: 'an other title' },
+  ],
+};
+
+URL.createObjectURL = () => '/blob/path/to/video';
+
+jest.mock('lib-components', () => ({
+  ...jest.requireActual('lib-components'),
+  useUploadManager: jest.fn(),
+}));
+const mockUseUploadManager = useUploadManager as jest.MockedFunction<
+  typeof useUploadManager
+>;
+
+const consoleError = jest
+  .spyOn(console, 'error')
+  .mockImplementation(() => jest.fn());
+
+describe('<VideoCreateForm />', () => {
+  const deferred = new Deferred();
```

Comment:
Defining it so globally is a little bit dangerous I think. You are reusing the same promise again and again. IMO this should be isolated for each test.